### PR TITLE
reduce crosslink batch and disable validator block broadcasting

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: node/node_handler.go [hunk 1]
```diff
@@ -360,7 +360,7 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block, commitSigAndBit
 		// 15% of the validator also need to do broadcasting
 		rand.Seed(time.Now().UTC().UnixNano())
 		rnd := rand.Intn(100)
-		if rnd < 1 {
+		if rnd < 0 {
```
Comment (denniswon) [inline node/node_handler.go]: so this is where we turn off for non-leaders

File: node/node_newblock.go [hunk 2]
```diff
@@ -127,7 +127,7 @@ func (node *Node) proposeNewBlock() (*types.Block, error) {
 
 	// Prepare cross links
 	var crossLinksToPropose types.CrossLinks
-	if node.NodeConfig.ShardID == 0 {
+	if node.NodeConfig.ShardID == 0 && node.Blockchain().Config().IsStaking(node.Worker.GetCurrentHeader().Epoch()) {
```
Comment (denniswon) [inline node/node_newblock.go]: and this is where we turn off for the leader "until the staking epoch"?
Is the intention to disable crosslink during the pre-staking period?
What is the reason for turning off the crosslink?

cc: @harmony-ek

File: node/node_handler.go (hunk 1 above)
Comment (rlan35) [reply node/node_handler.go]: yes

File: node/node_newblock.go (hunk 2 above)
Comment (rlan35) [reply node/node_newblock.go]: Cross link potentially is causing network trouble.

**Pull request merged**